package.json

```
@@ -31,15 +31,15 @@
     "@ember-data/rfc395-data": "^0.0.4"
   },
   "devDependencies": {
-    "babel-core": "^6.25.0",
+    "@babel/core": "^7.8.3",
     "lerna-changelog": "^0.8.3",
     "qunit": "^2.9.3",
     "qunit-eslint": "^2.0.0",
     "release-it": "^12.4.3",
     "release-it-lerna-changelog": "^1.0.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": ">= 12.*"
```

```suggestion
    "node": "10.* || >= 12.*"
```
It might worth it to adjust node versions here with ember-data’s

---

10.* LTS ended 10 days ago so I feel we should probably start life without it, @rwjblue thoughts?

---

@runspired I think you mean 8.* LTS (see https://nodejs.org/en/about/releases/)? Next Node version end of life (ie. 10.*) would be April 2021. We should be good for quite a long time after dropping Node 8.